config/seata-config-zk/src/main/java/io/seata/config/zk/ZookeeperConfiguration.java

```diff
@@ -47,7 +48,7 @@
 
     private static final String CONFIG_TYPE = "zk";
     private static final String ZK_PATH_SPLIT_CHAR = "/";
-    private static final String ROOT_PATH = ZK_PATH_SPLIT_CHAR + FILE_ROOT_CONFIG;
+    private static final String ROOT_PATH = ZK_PATH_SPLIT_CHAR + SEATA_FILE_ROOT_CONFIG;
```

Comment:
please check other Configuration Centers and assess if modifications are needed,such as etcd3.

Reply:
> please check other Configuration Centers and assess if modifications are needed,such as etcd3.

only zk